tests/test_RBAC.py

```diff
@@ -173,7 +174,7 @@ def test_compute_payload_without_additional_inputs(
     assert payload["dids"][0]["did"] == ddo.did
     assert payload["dids"][0]["serviceId"] == sa.id
     assert payload["algos"][0]["did"] == alg_ddo.did
-    assert payload["algos"][0]["serviceId"] == sa_compute.id
+    assert payload["algos"][0]["algorithmServiceId"] == sa_compute.id
```

Comment:
I have updated this tests too @calina-c . Is it OK?

Reply:
nope, I think we should keep the structure, it's just the population that differs. See there is already "algos" in the key, so it should be {"algos": {"did": algo did, "serviceId": algo service id}}, but the key inside remains "serviceId" for the RBAC request